### ISSUE TITLE
Reported segfault in SmartDashboard.putStringArray

### ISSUE THREAD

**Issue by virtuald** [robotpy/robotpy-wpilib#649]
/cc @celttechie 

Windows 10, Python 3.8.1 64-bit. Might be https://github.com/robotpy/robotpy-wpiutil/issues/7?

**Comment by auscompgeek** [comment]
Quite possibly if the strings are computed. I imagine constants would work by chance.

**Comment by auscompgeek** [comment]
Actually, hmm. ntcore stores a `std::vector<std::string>` and references that. vector and string own their contents, right?

**Comment by virtuald** [comment]
I looked at this a bit deeper, and I haven't been able to reproduce. I think if this were to happen, it would be that somehow an invalid py::str was returned from a function somewhere and added to an array that was used. Lots of pybind11 churn this summer, including a bunch of string stuff, so it's likely this has been addressed.

* https://github.com/pybind/pybind11/pull/2477
* https://github.com/pybind/pybind11/pull/2349
